pkg/controller/clusterdeployment/clusterdeployment_controller.go

```diff
@@ -637,19 +637,18 @@ func buildMasterMachineSet(clusterDeployment *clustop.ClusterDeployment, cluster
 		machineSet.Labels = make(map[string]string)
 	}
 	machineSet.Labels[clusterDeploymentLabel] = clusterDeployment.Name
-	machineSet.Labels[clustop.ClusterNameLabel] = cluster.Name
 	machineSet.OwnerReferences = []metav1.OwnerReference{*metav1.NewControllerRef(clusterDeployment, controllerKind)}
 	machineSetLabels := map[string]string{
-		machineSetNameLabel:      machineSet.Name,
-		clusterDeploymentLabel:   clusterDeployment.Name,
-		clustop.ClusterNameLabel: cluster.Name,
```

Comment:
Where did we land on keeping this as convention for ease of use on the command line? I know we mentioned a selector on the field but I can't spot that in the cluster-api PR. I do kind of like being able to see at a glance what's going on with labels.

Reply:
Yes, good catch. I did intend to add field selectors, but it slipped my mind. I will add them.